readme.md

```diff
@@ -0,0 +1,225 @@
+
+Port 1 is the coaxial one, Port 2 the microstrip port. A return loss greater than 23dB was achieved from DC to 20GHz.
+
+<p align = "center"><img src="./figures/plot_res_sim.png" alt="fig_plot_res_sim"  width="800"></p>
+<p align = "center">Figure 11: OpenEMS simulation result after optimization. Port 1 (coaxial), Port 2 (microstrip), Return Loss > 23dB</p>
+
+The smit chart plot in Figure 12 shows the in- and output trajectories on the complex reflection coefficient plane. They are nicely point symmetric to the origin. Overall these  results look promising. There is some margin for manufacturing tolerances left.
+
+The openEMS simulation takes about 20 minutes to complete.
+
+<p align = "center"><img src="./figures/plot_res_sim_smith.png" alt="fig_plot_res_sim_smith"  width="600"></p>
+<p align = "center">Figure 12: OpenEMS simulation result. In/ Output Matching from DC to 20GHz, Port 1 (coaxial), Port 2 (microstrip)</p>
+
+## 7. Measurement Results
+
+The fabricated test board from Figure 8 with two connectors back to back was measured with a high performance VNA (Vector Network Analyzer). The results are shown in Figure 13.
+
+<p align = "center"><img src="./figures/plot_res_meas_testboard.png" alt="fig_plot_res_meas_testboard  width="800"></p>
+<p align = "center">Figure 13: S-Parameter results, Test board Figure 8, two connectors back to back, reference plane at the SMA coax.</p>
+
+Let´s take a closer look at this plot. The S21 curve is almost perfect. It is flat with no dips. However, one will notice that the magnitude return loss curves are not identical. So, the structure is not entirely symmetrical. This is due to manufacturing tolerances. Keep in mind that we are talking about a low cost connector and PCB. The connectors are hand soldered, so some small displacements are to expected.
+
+In order to relate this result of Figure 13 to the simulated one in Figure 11 some some approximate observations can be done. Approximately, we can think about the measured structure as two identical imperfectly matched two ports (the connector) with low insertion loss and an ideal transmission line in between. Because the insertion loss of the two ports is low every miss match on either side will propagate directly through to the other side. The transmission line will further rotate the phase of the reflection coefficients with frequency. The 28.9mm transmission line is multiple wavelengths long at the highest frequency of interest (20GHz). So it is save to assume that the all possible phase angles between the two imperfectly matched connectors will occur over frequency.
```

Comment:
```suggestion
In order to relate this result of Figure 13 to the simulated one in Figure 11 some some approximate observations can be done. Approximately, we can think about the measured structure as two identical imperfectly matched two ports (the connector) with low insertion loss and an ideal transmission line in between. Because the insertion loss of the two ports is low, every miss match on either side will propagate directly through to the other side. The transmission line will further rotate the phase of the reflection coefficients with frequency. The 28.9mm transmission line is multiple wavelengths long at the highest frequency of interest (20GHz). So it is save to assume that the all possible phase angles between the two imperfectly matched connectors will occur over frequency.
```